ci/README.md

```diff
@@ -1,68 +0,0 @@
-# Deploy CI Infrastructure
```

Comment:
I think it's worth keeping notes about how to set up your CI around; maybe I missed a reference to it elsewhere?